static/js/themes.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const changeTheme = document.querySelector('#change-theme');
+let currentTheme = null;
+
+// Load themes
+for (let stylesheet of document.styleSheets) {
+	const rules = stylesheet.cssRules;
+	for (let rule of rules) {
+		const match = rule.selectorText?.match(/^\.theme-(\w+)$/)
+		if (match) {
+			const themeOption = document.createElement("option");
+			themeOption.value = rule.selectorText.substring(1);
```

Comment:
I don't like the substring. Add another group in the regex on line 10 to capture the whole selector after the dot and use that.